<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="mapper" >
<!--    <node pkg="plane_loc" type="mapper_bag_node" name="mapper" output="screen">-->
        <param name="settings" value="$(find plane_loc_2)/cfg/settings.yml" />
        <param name="objs_topic" value="/detector/objs" />
        <param name="acc_frames" value="5" />
        <param name="acc_duration" value="1.0e6" />
        <param name="merge_map_frames" value="10" />
<!--        <param name="acc_frames" value="15" />-->
<!--        <param name="acc_duration" value="2.0" />-->
<!--        <param name="merge_map_frames" value="30" />-->
<!--    </node>-->
    </group>
</launch>
